app/assets/javascripts/controllers/live_metrics/util/metrics-utils-factory.js

```diff
@@ -96,41 +96,59 @@ angular.module('miq.util').factory('metricsUtilsFactory', function() {
       dash.chartDataInit = true;
     }
 
-    var getContainerDashboardData = function(item, response) {
-      'use strict';
-      if (checkResponse(response) === false) {
-        return;
+    var timeTooltip = function (data) {
+      return '<div class="tooltip-inner">' +
+        moment(data[0].x).format('MM/DD hh:mm') + ' : ' +
+        data[0].value.toFixed(2) + '</div>';
+    };
+
+    var metricPrefix = function(maxValue) {
+      var m = {
+        prefix: '',
+        devider: 1
+      };
+
+      if (maxValue > Math.pow(10, 9)) {
+        m.prefix = 'G';
+        m.devider = Math.pow(10, 9);
+      } else if (maxValue > Math.pow(10, 6)) {
+        m.prefix = 'M';
+        m.devider = Math.pow(10, 6);
+      } else if (maxValue > Math.pow(10, 3)) {
+        m.prefix = 'k';
+        m.devider = Math.pow(10, 3);
       }
 
-      var data = response.data.data.sort(function(a, b) { return a.timestamp < b.timestamp; });
+      return m;
+    };
 
-      item.lastValues = {};
-      angular.forEach(data, function(d) {
-        item.lastValues[d.timestamp] = numeral(d.value).format('0,0.00a');
+    var getContainerDashboardData = function(item) {
+      'use strict';
+      item.data = item.data.sort(function(a, b) { return a.timestamp > b.timestamp; });
+      var maxValue = Math.max.apply(Math, item.data.map(function(o) { return o.value; }))
+      var m = metricPrefix(maxValue);
+
+      var id = 'ChartId-' + (btoa(item.id)).replace(/^[^a-z]+|[^\w:.-]+/gi, '');
```

Comment:
is this an elaborate way of removing the trailing `=` or is there something more behind this?